CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Eclipse GLSP Server Node Changelog
 
+## v1.1.0 - upcoming
+
+### Changes
+
+-   [graph] Align GGraph model with newest changes from glsp-server [#22](https://github.com/eclipse-glsp/glsp-server-node/pull/22) - Contributed on behalf of STMicroelectronics
```

Comment:
Please move this in the breaking changes section as Some gmodel classes/interfaces have been renamed.
Maybe also shortly note down the classes in question (e.g. `GLayoutContainer` (and related classes) => `GLayouting`))